packages/@aws-cdk/aws-lambda-nodejs/test/integ-handlers/dependencies-sdk-v3.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-console */
+// @ts-ignore
+import { S3 } from '@aws-sdk/client-s3'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved
```

Comment:
is there a S3 export in sdk js v3?

```suggestion
import { S3Client } from '@aws-sdk/client-s3'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved
```